cmd/plume/prerelease.go

```diff
@@ -127,6 +127,7 @@ func runFedoraPreRelease(system string, cmd *cobra.Command, args []string) error
 
 	spec := ChannelFedoraSpec()
 	ctx := context.Background()
+	client := http.Client{}
```

Comment:
Some of the changes in this commit don't seem relevant to the commit title.

Reply:
@arithx is this better now?

Reply:
Yep.